### PR TITLE
 Convert Gradle Build script to kts

### PR DESCRIPTION
**Pull request by jayashankar987**
migration changes from gradle build scripts to kotlin scripts

### CODE REVIEW

**Comment by jayashankar987** [comment]
https://github.com/adnan-SM/droidconChennaiV2/issues/3#issue-457892267

File: app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+dependencies {
+
+    implementation(fileTree("dir" to "libs", "include" to "*.jar"))
+    implementation("org.jetbrains.kotlin:kotlin-stdlib-jdk7:${Versions.kotlinVersion}")
+    implementation("androidx.appcompat:appcompat:1.0.0-beta01")
+    implementation("androidx.core:core-ktx:1.1.0-alpha05")
+    implementation("androidx.constraintlayout:constraintlayout:1.1.3")
+    testImplementation("junit:junit:4.12")
+    androidTestImplementation("androidx.test:runner:1.1.0-alpha4")
```
Comment (iamBedant) [inline app/build.gradle.kts]: Can we get these dependencies from `dependencies.kt`?

File: base/build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+    api(appDependencies.shimmer)
+    kapt(appDependencies.glide)
+    kapt(appDependencies.archCompiler)
+    api(firebaseDependencies.firestore)
+
+    implementation(fileTree("dir" to "libs", "include" to "*.jar"))
+
+    implementation("com.android.support:appcompat-v7:28.0.0-rc02")
+    testImplementation("junit:junit:4.12")
+    androidTestImplementation("com.android.support.test:runner:1.0.2")
```
Comment (iamBedant) [inline base/build.gradle.kts]: Can get these from `dependencies.kt` as well.

File: speakers/build.gradle.kts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+}
+
+dependencies {
+    implementation(fileTree("dir" to "libs", "include" to "*.jar"))
+
+    implementation("com.android.support:appcompat-v7:28.0.0-rc02")
+    testImplementation("junit:junit:4.12")
+    androidTestImplementation("com.android.support.test:runner:1.0.2")
+    androidTestImplementation("com.android.support.test.espresso:espresso-core:3.0.2")
```
Comment (RivuChk) [inline speakers/build.gradle.kts]: Move these to dependencies.kt?

File: build.gradle.kts [hunk 4]
```diff
@@ -1,33 +1,32 @@
...
         jcenter()
-        
+
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.2.0'
+        classpath("com.android.tools.build:gradle:3.3.2")
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:${Versions.kotlinVersion}")
```
Comment (iamBedant) [inline build.gradle.kts]: `dependencies.kt`

File: home/build.gradle.kts [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    }
+
+}
+
+dependencies {
+    implementation(fileTree("dir" to "libs", "include" to "*.jar"))
+
+    implementation("com.android.support:appcompat-v7:28.0.0")
+    testImplementation("junit:junit:4.12")
+    androidTestImplementation("com.android.support.test:runner:1.0.2")
```
Comment (iamBedant) [inline home/build.gradle.kts]: `dependencies.kt`

File: speakers/build.gradle.kts (hunk 3 above)
Comment (iamBedant) [inline speakers/build.gradle.kts]: `dependencies.kt`

File: buildSrc/src/main/kotlin/dependencies.kt [hunk 6]
```diff
@@ -0,0 +1,170 @@
+object Versions {
+
+    // framework config
+    val compileSdkVersion = 28
```
Comment (RivuChk) [inline buildSrc/src/main/kotlin/dependencies.kt]: use `const val` instead of just `val`?

File: build.gradle.kts [hunk 7]
```diff
@@ -1,33 +1,32 @@
...
         mavenCentral()
         jcenter()
-        
+
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.3.2'
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-        classpath 'com.google.gms:google-services:4.2.0'
+        classpath("com.android.tools.build:gradle:3.3.2")
```
Comment (RivuChk) [inline build.gradle.kts]: move to dependencies.kt?

**Review by RivuChk — CHANGES_REQUESTED** [review]

**Review by RivuChk — APPROVED** [review]

**Review by iamBedant — APPROVED** [review]

**Pull request merged**